modules/ocf/files/packages/cups/raster-filter

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# A Tea4CUPS filter to perform rasterization.
+# Based off the pdf-open script
+
+set -euo pipefail
+
+input=$(mktemp --suffix='.pdf')
+
+# Feed filter input into temporary file
+cat - > "$input"
+
+# If the file is not a PDF, then just pass it thorugh
+# (Note that all PDFs start with "%PDF")
+if [ "$(head -c 4 "$input")" != "%PDF" ]; then
```

Comment:
I'd use `file -bi "$input"` here to get the mimetype, and then check if it's `application/pdf`.

Reply:
Sounds good, will add another commit accordingly that checks for MIME types